group_vars/all/support.yml

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
I don't really understand why this is being done. The file contains nothing. This also changes the user interface, so I suggest that it remains as it was.